docs/src/components/motion/easing.module.css

```diff
@@ -153,9 +153,9 @@ b {
 }
 
 @media (width >= 1280px) {
-  .function {
+  /* .function {
```

Comment:
Let's kill this code. We typically want to avoid pushing code we aren't using to keep the files clean.

Reply:
Ah, yeah, I usually do too.  I forgot to clean this one up.